app/routes/forum/components/ForumList.css

```diff
@@ -0,0 +1,25 @@
+@import url('~app/styles/variables.css');
+
+.listEntry {
+  border-left: 8px solid var(--color-red-4);
```

Comment:
Be careful of using "random" red colors, considering red is our primary organization color. Ideally find something else